examples/embed/embed_themed.py

```diff
@@ -0,0 +1,60 @@
+import io
+
+from jinja2 import Template
+
+from bokeh.embed import components
+from bokeh.resources import INLINE
+from bokeh.util.browser import view
+from bokeh.io import curdoc, show
+from bokeh.themes import Theme
+from bokeh.plotting import figure
+
+x1 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+y1 = [0, 8, 2, 4, 6, 9, 5, 6, 25, 28, 4]
+
+p1 = figure(title='PINK')
+p1.scatter(x1, y1)
+
+theme = Theme(json={'attrs': {'Title': {'text_color': 'pink'}}})
```

Comment:
do you mean the background of the plot should match background-color for css class `embed-wrapper`?